torch/onnx/symbolic_opset9.py

```diff
@@ -1780,13 +1780,13 @@ def full(g, sizes, value, dtype, layout, device, pin_memory=False):
 
 def full_like(g, input, fill_value, dtype=None, layout=None, device=None, pin_memory=False, memory_format=None):
     fill_value = sym_help._maybe_get_const(fill_value, 'f')
+    dtype = sym_help._get_const(dtype, 'i', 'dtype')
+    dtype = 6 if dtype is None else dtype
```

Comment:
Is there an accompanying test for this specific update?

Reply:
test_prim_min also test this update since it uses full_like op